features/tracker-allowlist.json

```diff
@@ -409,22 +409,26 @@
             },
             "cloudflare.com": {
                 "rules": [
+                    {
+                        "rule": "cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.js",
```

Comment:
@GuiltyDolphin we should expand this rule to not block leaflet.js anywhere, as it's not a tracker and it's a common map component.

Looking at [other sites](https://raw.githubusercontent.com/duckduckgo/tracker-radar/main/domains/US/cloudflare.com.json) that fetch leaflet from the same endpoint, I see quite a bit of breakage (eg https://map.krak.dk/).

I think we should consider unblocking three.js everywhere on line 427 below as well, as it's a popular js library used for graphics and not a tracker.

Reply:
@dharb could you add some sites you've repro'd the breakage on to the issue, then we can generalise the mitigation